File: Week 1/kth smallest.py
n = int(input("Enter a size of an Array: "))
arr = []
for i in range(n):
    num = int(input())
    arr.append(num)
print(arr)

kth = int(input("which kth smallest you want?"))
arr.sort()
print("The kth smallest is ", arr[kth-1])


# another approch for sorting element
# my_list = [-15, -26, 15, 1, 23, -64, 23, 76]
# new_list = []

# while my_list:
#     min = my_list[0]  
#     for x in my_list: 
#         if x < min:
#             min = x
#     new_list.append(min)
#     my_list.remove(min)    

# print(new_list)

File: Week 1/kth smallest.cpp
#include<iostream>
#include<algorithm>
using namespace std;

void show(int arr[], int arr_size)
{
	for(int i;i<arr_size;i++)
	{
		cout<<arr[i]<< " ";
	}
}
int main()
{
	int num,arr[num],k;
	cout<<"Enter the Size of an Array:";
	cin>> num;
	cout<<"Enter the Elements in an Array:"<<endl;
	for(int i=0;i<num;i++)
	{
		cin>>arr[i];
	}
	cout<<"The Array is ";
	show(arr,num);
	cout<<endl;
	sort(arr,arr+num);
	cout<<"Which kth Smallest Number You Want? "<<endl;
	cin>>k;
	cout<<"The "<<k<<"th Smallest Element is "<<arr[k-1];
	
}
